dev/com.ibm.ws.classloading/resources/OSGI-INF/l10n/library.properties

```
@@ -59,6 +62,11 @@ folder.name=Folder
 folder.name.desc=Folder containing resources such as .properties and .class files
 folder.dir=Folder
 folder.dir.desc=Directory or folder to be included in the library classpath for locating resource files
+#path element
+path=Path
+path.desc=Add a path to be included in this library
+path.name=Name
+path.name.desc= Path to be included in the library classpath for locating resource files
```

what does "for locating resource files" modify here? "Path" or "library classpath"?  If the latter..
```suggestion
path.name.desc= Path to include in the library classpath that locates resource files
```